src/library_html5.js

```diff
@@ -1865,6 +1865,12 @@ var LibraryJSEvents = {
     return GL.currentContext ? GL.currentContext.handle : 0;
   },
 
+  emscripten_webgl_get_drawing_buffer_size: function(contextHandle, width, height) {
+    var GLContext = GL.getContext(contextHandle).GLctx;
```

Comment:
If `GL.getContext(contextHandle)` returns null, or if `GL.getContext(contextHandle).GLctx` is not a valid context, then this function should return `{{{ cDefine('EMSCRIPTEN_RESULT_INVALID_PARAM') }}}` and not abort execution by throwing.